### PR TITLE
Add OpenFeature providers to SDK reference

### PR DESCRIPTION
**Pull request by z4kn4fein**
### Description

This PR adds a new `OpenFeature Providers` section under the `SDK References` menu that lists and documents all the ConfigCat providers.

### Notes for QA

- Docs -> SDKs -> SDK References -> OpenFeature Providers

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [x] I have added my changes to the V1 and V2 documentations.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/versioned_docs/version-V1/sdk-reference/openfeature/python.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+if is_awesome_feature_enabled:
+    do_the_new_thing()
+else:
+    do_the_old_thing()
+```
+
+## Evaluation Context
+
+An <a href="https://openfeature.dev/docs/reference/concepts/evaluation-context" target="_blank">evaluation context</a> in the OpenFeature specification is a container for arbitrary contextual data that can be used as a basis for feature flag evaluation.
+The ConfigCat provider translates these evaluation contexts to ConfigCat [User Objects](../../php/#user-object).
```
Comment (laliconfigcat) [inline website/versioned_docs/version-V1/sdk-reference/openfeature/python.mdx:61]:
```suggestion
The ConfigCat provider translates these evaluation contexts to ConfigCat [User Objects](../../python/#user-object).
```

File: website/docs/sdk-reference/openfeature/python.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+if is_awesome_feature_enabled:
+    do_the_new_thing()
+else:
+    do_the_old_thing()
+```
+
+## Evaluation Context
+
+An <a href="https://openfeature.dev/docs/reference/concepts/evaluation-context" target="_blank">evaluation context</a> in the OpenFeature specification is a container for arbitrary contextual data that can be used as a basis for feature flag evaluation.
+The ConfigCat provider translates these evaluation contexts to ConfigCat [User Objects](../../php/#user-object).
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/openfeature/python.mdx:61]:
```suggestion
The ConfigCat provider translates these evaluation contexts to ConfigCat [User Objects](../../python/#user-object).
```

**Review by laliconfigcat — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-V1/sdk-reference/openfeature/js.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+```
+
+</TabItem>
+<TabItem value="CDN" label="CDN">
+
+```html
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/@openfeature/config-cat-web-provider/index.cjs.min.js"
+></script>
```
[the remark this answers is not in the document]
Comment (z4kn4fein) [reply website/versioned_docs/version-V1/sdk-reference/openfeature/js.mdx]: Done

**Review by laliconfigcat — APPROVED** [review]

File: website/versioned_docs/version-V1/sdk-reference/android.mdx [hunk 4]
```diff
@@ -265,8 +265,8 @@ User user = User.newBuilder().build("john@example.com");
 
 ```java
 java.util.Map<String,String> customAttributes = new java.util.HashMap<String,String>();
-    customAttributes.put("SubscriptionType", "Pro");
-    customAttributes.put("UserRole", "Admin");
+customAttributes.put("SubscriptionType", "Pro");
```
[the remark this answers is not in the document]
Comment (z4kn4fein) [reply website/versioned_docs/version-V1/sdk-reference/android.mdx:268]: Heh, they were there, I just didn't hit the save button 🤦

**Pull request merged**